### DISCUSSION TITLE
Drucklayout - Strichstärke einer Linie breiter machen

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#175] [Q&A]
Nachfolgend die Definition eines Rechteckes im Drucklayout (der rote Umriss soll das Symbol eines Geltungsbereiches darstellen)
Der Umriss soll durch eine breite Linie repräsentiert werden:

``` xml
<panel dock="right" width="40">			
		<text string="Geltungsbereich" align="left" x="100" y="10" fontsize="2.0" font="Verdana" fontcolor="0,0,0"/>			
		<line x1="1" y1="2" x2="30" y2="2" color="255,255,0" width="12"/>
		<line x1="1" y1="2" x2="1" y2="8" color="255,0,0" width="12"/>
		<line x1="1" y1="8" x2="30" y2="8" color="255,0,0" width="12"/>
		<line x1="30" y1="8" x2="30" y2="2" color="255,0,0" width="12"/>	 
</panel>
```

Allerdings hat der Eintrag "width=xx" keine Auswirkung auf die Linienbreite.
Gibt es da einen anderen key?

Wenn das mit den Linien nicht machen kann: gibt es eine andere Variante?

**Comment by gis-eni** [comment]
Ich weiß nicht, woher du das mit ``width`` hast, weil für Linien sind nur die Attrubte ``x1, y1, x2, y2, color`` implementiert.
Werde das einmal als Verbesserungsvorschlag markieren...
